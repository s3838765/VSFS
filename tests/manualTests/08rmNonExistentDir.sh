#! /usr/bin/sh
../../VSFS rm 08nonExistentDir.notes 08nonExistentDir/ 1> 08nonExistentDir.out 2>&1
