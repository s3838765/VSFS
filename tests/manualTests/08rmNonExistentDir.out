A directory name was provided. Please use the rmdir command to remove directories. Terminating program.
